docker-files/zeeguu-web/apache-zeeguu.conf.default

```diff
@@ -54,8 +64,6 @@ ServerName localhost
 
     Alias /static /var/www/zeeguu-web/zeeguu_web/static
 
-    Alias /api/resources /var/www/static
```

Comment:
How are the news-icons exposed in this case? For me it still tries to fetch them from ``/api/resources/<icon_name>``

Reply:
The latest version of the reader has the icons deployed within it's own static folder and the latest version of the Core/API returns only a icon_name instead of an URL. Make sure that you get the latest version of the submodules.